recipe/meta.yaml

```diff
@@ -1,16 +1,36 @@
 {% set name = "autogluon" %}
-{% set version = "1.0.0" %}
+{% set version = "1.1.1" %}
+
+{% set boto3 = ">=1.10,<2" %}
+{% set numpy_local = ">=1.21,<1.29" %}
+{% set pandas_local =  ">=2.0.0,<2.3.0" %}
+{% set scikit_learn = ">=1.3.0,<1.4.1" %}
+{% set scipy = ">=1.5.4,<1.13" %}
+{% set psutil = ">=5.7.3,<6" %}
+{% set s3fs = ">=2023.1,<2025" %}
+{% set networkx = ">=3.0,<4" %}
+{% set tqdm = ">=4.38,<5" %}
+{% set Pillow = ">=10.0.1,<11" %}
+{% set pytorch = ">=2.2,<2.4" %}
+# This pinning is used for both lightning and pytorch-lightning
+# Do not attempt to replace one library with another as this causes bugs/test failures
+{% set lightning = ">=2.2,<2.4" %}
+{% set async_timeout = ">=4.0,<5" %}
```

Comment:
```suggestion
{% set networkx = ">=3.0,<4" %}
{% set tqdm = ">=4.38,<5" %}
{% set Pillow = ">=10.0.1,<11" %}
{% set pytorch = ">=2.2,<2.4" %}
# This pinning is used for both lightning and pytorch-lightning
# Do not attempt to replace one library with another as this causes bugs/test failures
{% set lightning = ">=2.2,<2.4" %}
```

The `s3fs` and `async_timeout` variables are unused. Should they be included somewhere? If not, we can remove these variables.